test/e2e/frictionless/features/regression/converter.feature

```diff
@@ -82,4 +82,39 @@ Feature: Frictionless Converter Block
 
   Examples:
       | Verb     | File                |
-      | sign-pdf | test-files/test.pdf |
\ No newline at end of file
+      | sign-pdf | test-files/test.pdf |
+
+  @MWPW-135907 @regression @converter
```

Comment:
Please replace the tag with @MWPW-137180.

Reply:
I fixed this, can you please check?